bin/steps/collectstatic

```diff
@@ -42,6 +46,22 @@ if [ ! "$DISABLE_COLLECTSTATIC" ] && [ -f "$MANAGE_FILE" ] && [ "$DJANGO_INSTALL
 
         mcount "failure.collectstatic"
 
+        if grep -q 'SyntaxError' "$COLLECTSTATIC_LOG"; then
+            mcount "failure.collectstatic.syntax-error"
```

Comment:
You might want to turn these into `else if`s if they are mutually exclusive

Reply:
they aren't.

Reply:
python can have multiple tracebacks